apps/react-18-tests-v8/cypress-react-18.config.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path';
+import { baseConfig, baseWebpackConfig } from '@fluentui/scripts-cypress';
+import { createStorybookWebpackConfig } from '@fluentui/scripts-webpack';
+
+// Include all tests from this app and the components package
+const specs = [path.resolve('src/**/*.cy.{tsx,ts}'), path.resolve('../../packages/react-examples/**/*.e2e.{tsx,ts}')];
+
+const config = { ...baseConfig };
+
+config.component.specPattern = specs;
+config.component.devServer.webpackConfig = createStorybookWebpackConfig(baseWebpackConfig);
```

Comment:
`createStorybookWebpackConfig` is used within react-examples to build webpack config, I added it here as well for simplicity of not introducing manual config